UI/ui_craft.csv

```diff
@@ -3,35 +3,35 @@ TodaySuccess#0,"""Realizados hoy"""
 NextReward#0,"""Siguiente recompensa"""
 AcceptList#0,"""Solicitudes"""
 OrderReward#0,"""Recompensas"""
-Recipe#0,"""対応レシピ"""
-SuccessNum#0,"""依頼達成回数"""
-SuccessReward#0,"""recompensas"""
+Recipe#0,"""Receta"""
+SuccessNum#0,"""Numero de solicitudes completadas"""
+SuccessReward#0,"""Recompensas"""
 FullSizeFormTitle#0,"""Craft diario"""
 FullSizeFormHelp#0,"""<c f0df60>You receive various craft requests daily. Selecciona un request to carry out the crafting.<c>"""
 OrderListEmptyMessage#0,"""No hay solicitudes"""
-OrderSuccessNumMessage#0,"""$(0)/$(1)\u3000(Renovación diaria a las $(2):00)"""
-RewardListItemName#0,"""$(0)依頼達成"""
+OrderSuccessNumMessage#0,"""Hechos $(0)/$(1)\u3000( Renovación diaria a las $(2):00 )"""
+RewardListItemName#0,"""$(0) Solicitudº"""
```

Comment:
once again, PSO2 can not display `º`, please remove it

Reply:
I could not find what was wrong >.<

Sorry

Reply:
`./UI/ui_craft.csv	13	39	0xba	MASCULINE ORDINAL INDICATOR`

click on the red X, it will lead you to the buildbot page
for "Verify Font rendering" step like your was, it give you the file name, line number, line position, the hex version (we can not print SOME characters then the unicode name of said char 
`./PATH/FILE.CSV	line#	pos#	hex	Charname`